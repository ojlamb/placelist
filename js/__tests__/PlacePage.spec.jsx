import React from "react";
import { shallow } from "enzyme";
import Places from "../components/places/PlacesPage";

test("Places renders correctly", () => {
  const component = shallow(<Places />);
  expect(component).toMatchSnapshot();
});
