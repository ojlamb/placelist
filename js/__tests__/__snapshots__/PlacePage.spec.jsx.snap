// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places renders correctly 1`] = `
<Connect(PlacePage)
  classes={
    Object {
      "actionButton": "Connect-PlacePage--actionButton-1",
      "cardGrid": "Connect-PlacePage--cardGrid-4",
      "icon": "Connect-PlacePage--icon-2",
      "layout": "Connect-PlacePage--layout-3",
    }
  }
/>
`;
